.gitignore

```diff
@@ -62,3 +62,13 @@ benchmarks/bench_covertype_data/
 !*/src/*.cpp
 *.sln
 *.pyproj
+
+sklearn/sklearn.egg-info/top_level.txt
```

Comment:
No one should be building within the sklearn directory. Please revert these changes.

Reply:
Ok, I have reverted this change to the ``.gitignore`` file